public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.tsx

```diff
@@ -386,19 +386,6 @@ export const DataConfigPanelItem = ({
               )}
               {/* Show input fields for dimensions */}
               {!isAggregations && getCommonDimensionsField(selectedObj, name)}
-              {isPositionButtonVisible(name) && (
-                <EuiFormRow label="Side">
-                  <ButtonGroupItem
-                    legend="Side"
-                    groupOptions={[
-                      { id: 'left', label: 'Left' },
-                      { id: 'right', label: 'Right' },
-                    ]}
-                    idSelected={selectedObj.side || 'right'}
-                    handleButtonChange={(id: string) => updateList(id, 'side')}
-                  />
-                </EuiFormRow>
-              )}
```

Comment:
why is this being removed?

Reply:
This belongs to a work that was way outdated since 2.4, it is not even working and needed.